### PR TITLE
Reproduced issue #1779

### PR DESCRIPTION
**Pull request by Naktibalda**
Failing test for issue #1779

### CODE REVIEW

**Comment by Naktibalda** [comment]
The simplest possible fix.

**Comment by samdark** [comment]
Looks valid.

**Comment by Naktibalda** [comment]
Actually getClassesFromFile shouldn't include, it violates SRP.

File: src/Codeception/Lib/Parser.php [hunk 1]
```diff
@@ -101,7 +101,7 @@ protected function addCommentStep($comment)
 
     public static function getClassesFromFile($file)
     {
-        include_once $file;
+        self::includeFile($file);
```
Comment (DavertMik) [inline src/Codeception/Lib/Parser.php]: I think anonymous function can be better here. At least it does just what needed - creates a new context without implementing a new method

Comment (Naktibalda) [inline src/Codeception/Lib/Parser.php]: But it would look much uglier and be much slower.

**Comment by DavertMik** [comment]
Ok, thanks!

**Pull request merged**